docs/Guides/Plugins-Guide.md

```diff
@@ -497,6 +497,15 @@ warning.create('FastifyDeprecation', 'FST_ERROR_CODE', 'message')
 warning.emit('FST_ERROR_CODE')
 ```
 
+If instead you want to disable all the warnings emitted by Fastify, you can
+do it by:
+
+- setting the `NODE_NO_WARNINGS` environment variable to `1`
+- passing the `--no-warnings` flag to the node process
+- setting 'no-warnings' in the `NODE_OPTIONS` environment variable
+
+For more information see [node's documentation](https://nodejs.org/api/cli.html).
```

Comment:
My bad, I didn't notice the Emit Warning title but just the `process-warnings` import in the code example.
Should I create another paragraph called "Suppress warnings"?

Reply:
I'm not clear if this document is the right place for the documentation. Also, are _all_ of the listed items required to accomplish the goal? Or are they different ways to accomplish the same goal?

Reply:
The doc modified in this PR is about how to write plugins. The documentation being added is about how to deal with warnings emitted by Fastify. These are different goals. The added documentation is not a good fit for the document modified in this PR. It may require its own document if an existing one is not a good fit.